welcome.css

```diff
@@ -74,4 +74,89 @@ html, body {
   text-align: center;
   font-size: 32px;
   color: olive;
-}
\ No newline at end of file
+}
+
+.fa-bars::before {
```

Comment:
what is the meaning of `fa` ?
I can not understand the meaning of `.fa-bars`
If you wonder which name you should use, just ask Eight.

Reply:
I didn’t choose the "fa-bars",  I take the three lines or hamburger icon from google maybe it's the icon name.
but i gave a class name "burger-icon"